src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let total = days * 40;
```

Comment:
Consider using a constant for the daily rental cost (40) to make the code more maintainable and easier to understand.